### PR TITLE
Explains how to check TLS version and ciphers

### PR DESCRIPTION
**Pull request by cadmiumcat**
What
----
Explains how to find out about TLS version and ciphers

How to review
-------------

1. Preview the content locally ([see README](https://github.com/alphagov/paas-tech-docs#preview)) and check that it renders as expected.
1. Manually check that any removed or renamed anchor tags are not in use ([linkchecker](https://github.com/linkcheck/linkchecker) doesn't do this).
1. Read and check that the new info makes sense

Who can review
--------------

Anyone

### CODE REVIEW

File: source/documentation/deploying_services/use_a_custom_domain.erb [hunk 1]
```diff
@@ -450,6 +450,10 @@ The above example configures the CDN route to have a default time-to-live of 15
 
 The CDN route service forwards cookie headers your app by default, so cookie-based authentication will work as expected. Other headers, such as HTTP auth, are stripped by default. If you need a different configuration, see the guidance on [forwarding headers](/deploying_services/use_a_custom_domain/#forwarding-headers).
 
+### Verifying protocols and cyphers
+
+You can find the TLS version and cyphers used on GOV.UK PaaS by using an SSL testing tool such as [SSL Labs](https://www.ssllabs.com/ssltest/). For your apps, check the hostname `cloudapps.digital` (or `london.cloudapps.digital` in the London region).
```
Comment (AP-Hunt) [inline source/documentation/deploying_services/use_a_custom_domain.erb:455]: This isn't quite right. Looking at the TLS ciphers on `(london.)cloudapps.digital` will show you those in use by our ALB. You need to test it against your custom domain with CloudFront to find the ones being used in communication with the client.

Comment (cadmiumcat) [reply source/documentation/deploying_services/use_a_custom_domain.erb:455]: Is that also the case for the TLS version? 🤔

Comment (AP-Hunt) [reply source/documentation/deploying_services/use_a_custom_domain.erb:455]: Yes

File: source/documentation/deploying_services/use_a_custom_domain.erb [hunk 2]
```diff
@@ -450,6 +450,10 @@ The above example configures the CDN route to have a default time-to-live of 15
 
 The CDN route service forwards cookie headers your app by default, so cookie-based authentication will work as expected. Other headers, such as HTTP auth, are stripped by default. If you need a different configuration, see the guidance on [forwarding headers](/deploying_services/use_a_custom_domain/#forwarding-headers).
 
+### Verifying protocols and cyphers
+
+You can find the TLS version and cyphers used on GOV.UK PaaS by using an SSL testing tool such as [SSL Labs](https://www.ssllabs.com/ssltest/). For your apps, use your CloudFront custom domain as the hostname.
```
Comment (AP-Hunt) [inline source/documentation/deploying_services/use_a_custom_domain.erb:455]: s/use your CloudFront custom domain/use your custom domain/

**Review by AP-Hunt — CHANGES_REQUESTED** [review]
I think adding the word CloudFront here introduces some possible confusion between the tenant's domain name, and the host name of the CloudFront distribution.

File: source/documentation/deploying_services/use_a_custom_domain.erb (hunk 2 above)
Comment (cadmiumcat) [reply source/documentation/deploying_services/use_a_custom_domain.erb:455]: ah yes, I can see that

**Review by AP-Hunt — APPROVED** [review]

**Pull request merged**